README.md

```diff
@@ -92,7 +92,11 @@ You can start a new Camel K integration with or without additional options such
 
 * **Dependencies - Apache Camel K Integration with Explicit Dependencies**
 
-    Specify dependency details either by their camel-component artifact Id or by their Maven coordinates (group:artifact:version), with the option to specify more than one. For more information about configuration with Dependencies, see [Dependencies and Component Resolution](https://camel.apache.org/camel-k/latest/configuration/dependencies.html) in the Apache Camel-K documentation.
+    Specify dependency details either by their camel-component artifact Id or by their Maven coordinates (group:artifact:version), with the option to specify more than one. For more information about configuration with Dependencies, see [Dependencies and Component Resolution](https://camel.apache.org/camel-k/latest/configuration/dependencies.html) in the Apache Camel K documentation.
+
+* **Use a predefined Task - useful for multi-attributes deployment**
+
+    Allow to use a predefined Task of type "camel-k". It is useful when deploying an Apache Camel K integration which requires several parameters. To provide a task, see section [Creating a new Camel K Integration with multiple parameters](#Creating-a-new-Camel-K-Integration-with-multiple-parameters) 
```

Comment:
I might change the language in this area slightly.

* **Define a Task for Easily Reproducible Multi-attribute Deployment**

Some types of deployments require more complex combinations of attributes. In these cases, defining a custom Task of type "camel-k" is useful. To create a task, see section [Creating a new Camel K Integration with multiple parameters](#Creating-a-new-Camel-K-Integration-with-multiple-parameters)